src/vma/proto/dst_entry_udp_mc.cpp

```diff
@@ -81,6 +81,24 @@ bool dst_entry_udp_mc::conf_l2_hdr_and_snd_wqe_ib()
 	return ret_val;
 }
 
+void dst_entry_udp_mc::set_src_addr()
+{
+	m_pkt_src_ip = INADDR_ANY;
+	
+	if (m_bound_ip) {
+		m_pkt_src_ip = m_bound_ip;
```

Comment:
Following bind() m_pkt_src_ip will be updated. Is it also applicable to UC case? If not - please justify